src/ui/default-prop-types.js

```diff
@@ -59,7 +59,8 @@ export const settingsShape = PropTypes.shape({
   homeLanguages: PropTypes.arrayOf(homeLanguageShape).isRequired,
   helpPage: PropTypes.objectOf(settingsDocumentShape).isRequired,
   termsPage: PropTypes.objectOf(settingsDocumentShape).isRequired,
-  footerLinks: PropTypes.objectOf(PropTypes.arrayOf(settingsDocumentShape)).isRequired
+  footerLinks: PropTypes.objectOf(PropTypes.arrayOf(settingsDocumentShape)).isRequired,
+  defaultTags: PropTypes.arrayOf(PropTypes.string)
```

Comment:
Wouldn't it be something we expect to be there (At least as an empty array)?

Reply:
Now that I pass it to the component as `defaultTags={settings.defaultTags || []}` it should be required here, true.
Corrected.